content/source/docs/cloud/registry/index.html.md

```diff
@@ -2,13 +2,21 @@
 layout: "cloud"
 page_title: "Private Module Registry - Terraform Cloud and Terraform Enterprise"
 description: |-
-  Use the Terraform Cloud private module registry to share Terraform modules across your organization. 
+  Use the Terraform Cloud private module registry to share Terraform modules across your organization.
 ---
 
 # Private Module Registry
 
+> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
+
 > **Hands-on:** Try the [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
 
 Terraform Cloud's private module registry helps you share [Terraform modules](/docs/language/modules/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a configuration designer to help you build new workspaces faster.
 
 By design, the private module registry works much like the [public Terraform Registry](/docs/registry/index.html). If you're already used the public registry, Terraform Cloud's registry will feel familiar.
+
+The private module registry lets you add both private and public modules to the registry, so that you can more easily find all your modules in one place. Private modules are modules that you publish directly to the registry. Public modules, also referred to as "publicly curated modules", live in the public Terraform Registry and are synchronized from it. By adding some of the many public Terraform Registry modules to your private module registry, you can maintain a curated list of approved modules for your organization.
+
+-> **Note:** Publicly curated modules are not supported in Terraform Enterprise.
+
+You can even use a [Sentinel Policy](/docs/cloud/sentinel/index.html) to mandate that all non-root modules used in Terraform configurations be private or publicly curated modules from your own private module registry. You can even require that all those modules use recent versions with a policy like [use-recent-versions-from-pmr.sentinel](https://github.com/hashicorp/terraform-guides/blob/master/governance/third-generation/cloud-agnostic/http-examples/use-recent-versions-from-pmr.sentinel).
```

Comment:
Thank you so much for adding this - it's great!! I'm just reorganizing it a tiny bit and adding some links to other pages. See what you think!

```suggestion
Terraform Cloud's private module registry works similarly to the [public Terraform Registry](/docs/registry/index.html) and helps you share [Terraform modules](/docs/language/modules/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a configuration designer to help you build new workspaces faster.  

You can add both private and public modules to the registry:

- [Private modules](/docs/cloud/registry/publish.html) directly to the registry are hosted on the registry and are only available to members of that organization. In Terraform Enterprise, they are also available to organizations that are configured to [share modules](/docs/enterprise/admin/module-sharing.html) with that organization. 
- [Public modules](/docs/cloud/registry/add.html), are automatically synchronized from the Terraform Registry where they are hosted. Public modules are not supported in Terraform Enterprise.

You can use [Sentinel Policies](/docs/cloud/sentinel/index.html) to enforce rules about how members of your organization can use these modules. For example, you can mandate that all non-root modules in Terraform configurations be private or public modules from your own private module registry. You can even require that all of those modules use recent versions with a policy like [use-recent-versions-from-pmr.sentinel](https://github.com/hashicorp/terraform-guides/blob/master/governance/third-generation/cloud-agnostic/http-examples/use-recent-versions-from-pmr.sentinel).
```

Reply:
I'm fine with the changes, but think you need some slight changes onl your new lines 18 and 19.  Line 18 seems to need a verb like "are added" after the link.  And line 19 should remove the comma after the link.